redis_exporter.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: redis_exporter
```

Comment:
might it be helpful to call this `prometheus-redis_exporter`?

Reply:
the upstream has redis_exporter all over , just wanted to keep the same parity , happy to change it